src/components/NewBoardForm.jsx

```diff
@@ -5,9 +5,13 @@ import {
 } from '@material-ui/core'
 import { useMutation } from '@apollo/client'
 import { ADD_BOARD } from '../graphql/mutations'
+import { GET_ALL_BOARDS } from '../graphql/queries'
 
 const NewBoardForm = ({ setOpen, open }) => {
-    const [addBoard] = useMutation(ADD_BOARD)
+    const [addBoard] = useMutation(ADD_BOARD, {
```

Comment:
Are [] needed?

Reply:
they are, useMutation returns an array of stuff.